hack/ci/operator-ensure-manifests.sh

```diff
@@ -13,30 +13,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Script responsible for ensuring and correcting manifests as needed.
+set -e
 
-source ./hack/env.sh
+source ./hack/ci/ensure-image.sh
 
-OUTPUT="${PWD}/build/_output/operatorhub"
+tempfolder=$(mktemp -d)
+echo "$tempfolder"
 
-echo "---> Output dir is set to ${OUTPUT}"
-# clean up
-rm -rf "${OUTPUT}"
+version=$(getOperatorVersion)
+latest_released_version=$(getLatestOlmReleaseVersion)
 
-mkdir -p "${OUTPUT}"
+git clone https://github.com/operator-framework/community-operators.git ${tempfolder}
+mkdir  "${tempfolder}/community-operators/kogito-operator/${version}/"
+cp -r bundle/manifests/*.yaml "${tempfolder}/community-operators/kogito-operator/${version}/"
 
-# copying all the kogito dependent operator to be added at manifests root.
-rm -rf ~/operators/
-git clone https://github.com/operator-framework/community-operators.git ~/operators/
-cp -r ~/operators/community-operators/strimzi-kafka-operator "${OUTPUT}"
-cp -r ~/operators/community-operators/keycloak-operator "${OUTPUT}"
-cp -r ~/operators/community-operators/infinispan/ "${OUTPUT}"
-cp -r ~/operators/community-operators/kogito-operator/ "${OUTPUT}"
-rm -rf ~/operators/
 
-cp -r "deploy/olm-catalog/kogito-operator/$(getOperatorVersion)/" "${OUTPUT}/kogito-operator/"
-cp "deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml"  "${OUTPUT}/kogito-operator/"
+#replace image in target CSV
+sed -i  "s|quay.io/kiegroup/kogito-cloud-operator.*|kind-registry:5000/kiegroup/kogito-cloud-operator|g"  "${tempfolder}/community-operators/kogito-operator/${version}/kogito-operator.clusterserviceversion.yaml"
+#
+##ensure correct replace field is there
+sed -i "s|replace.*|replaces: kogito-operator.v${latest_released_version}|g" "${tempfolder}/community-operators/kogito-operator/${version}/kogito-operator.clusterserviceversion.yaml"
+#
+sed -i "s|${latest_released_version}|${version}|g" "${tempfolder}/community-operators/kogito-operator/kogito-operator.package.yaml"
+#
+echo "---> verify CSV updates"
+cat "${tempfolder}/community-operators/kogito-operator/${version}/kogito-operator.clusterserviceversion.yaml" | grep kind-registry:5000/kiegroup/kogito-cloud-operator
```

Comment:
I see `kind-registry:5000/kiegroup/kogito-cloud-operator` in many scripts ... maybe should we have it in env ?
that would avoid to update all scripts if there is a change...

Reply:
done